frontend/public/components/daemon-set.jsx

```diff
@@ -56,12 +58,14 @@ const DaemonSetTableHeader = () => {
       title: 'Status', sortFunc: 'daemonsetNumScheduled', transforms: [sortable],
       props: { className: tableColumnClasses[3] },
     },
-    {
-      title: 'Pod Selector', sortField: 'spec.selector', transforms: [sortable],
+    { title: 'Owner', sortField: 'spec.selector', transforms: [sortable],
```

Comment:
Need to update `sortField` here. You'll need a custom `sortFunc` for owner references. Here is an example:

https://github.com/openshift/console/blob/master/frontend/public/components/factory/table.tsx#L110-L113

Same comment for other tables below